draft-ietf-idr-5g-edge-service-metadata-24.xml

```diff
@@ -973,14 +1066,27 @@
         <t> - AS Recognition: The router will check the AS value in the AS-Scope Sub-TLV.</t>
 
         <t> - If the AS value matches the local AS or a recognized AS in its configuration, the router will process the update as usual. If the AS value does not match or is not recognized, the router MUST drop the BGP UPDATE message containing the AS-Scope Sub-TLV.</t>
+	<!-- XXX JMH the desired value here is RFC 7606 style "treat as
+	withdraw", not "drop the update" -->
 		
+	<!-- XXX JMH - I think you can drop the forwaring rules section. The
+	motivation is explained previously.
+
+	Your TLVs don't have transitivity in them and the entire containing
+	metadata path attribute is non-transitive.  I think you can drop the
+	next sentence as well.
+
+	I'm leaving the text in place in case your intent is different than
+	those two observations.
+
```

Comment:
The intent is to say: 

"If the AS value does not match or is not recognized, the router SHOULD NOT process the Metadata Path Attribute values in the BGP UDPATE and SHOULD NOT propagate the received BGP UPDATE to other nodes".

Reply:
I think that means the text that isn't commented out accomplishes that goal.  Is it then okay to delete the commented out text?

Reply:
yes.

Reply:
Removed commented out text and resolving this item.